README.md

```diff
@@ -100,15 +100,15 @@ Once the service is created, create new credentials by selecting `Service Creden
 From Watson Studio project that you have created earlier, go to `+ Add to Project` and choose `Connection` 
 ![](doc/source/images/db2-3.png)
 
-Select `Db2 Warehouse` from the available options to connect to Db2 warehouse you created earlier.
+Select `Db2 Warehouse` from the available options to connect to Db2 warehouse on cloud you created earlier.
 ![](doc/source/images/db2-4.png)
 
-Configure the connection based on the DB2 credentials you saved earlier.
+Configure the connection based on the Db2 credentials you saved earlier.
 ![](doc/source/images/db2-5.png)
 
-### 4. Create and load data to DB2 warehouse
+### 4. Create and load data into a Db2 Warehouse on Cloud database
```

Comment:
I think the link will be broken, because the words must match. Above you have:
```#4-create-and-load-data-to-db2-warehouse-on-cloud-database```
here you have `into` instead of `to`

Reply:
Change the "to" to "into a"